assets/js/src/newsletter_editor/blocks/products.js

```diff
@@ -0,0 +1,607 @@
+/**
+ * Posts block.
+ *
+ * This block works slightly differently compared to any other block.
+ * The difference is that once the user changes settings of this block,
+ * it will be removed and replaced with other blocks. So this block will
+ * not appear in the final JSON structure, it serves only as a placeholder
+ * for posts, that will be comprised of container, image, button and text blocks
+ *
+ * This block depends on blocks.button and blocks.divider for block model and
+ * block settings view.
+ */
+import Backbone from 'backbone';
+import Marionette from 'backbone.marionette';
+import Radio from 'backbone.radio';
+import _ from 'underscore';
+import jQuery from 'jquery';
+import MailPoet from 'mailpoet';
+import App from 'newsletter_editor/App';
+import CommunicationComponent from 'newsletter_editor/components/communication';
+import BaseBlock from 'newsletter_editor/blocks/base';
+import ButtonBlock from 'newsletter_editor/blocks/button';
+import DividerBlock from 'newsletter_editor/blocks/divider';
+import 'select2';
+
+var Module = {};
+var base = BaseBlock;
+var PostsDisplayOptionsSettingsView;
+var SinglePostSelectionSettingsView;
+var EmptyPostSelectionSettingsView;
+var PostSelectionSettingsView;
+var PostsSelectionCollectionView;
+
+Module.PostsBlockModel = base.BlockModel.extend({
```

Comment:
it is basically just a copy and further modifications will come in upcoming tasks but I think we should rename the Model and views so that it is clear that it is `ProductBlock`